### ISSUE TITLE
Build tools

### ISSUE THREAD

**Issue by Chris00** [ocaml/v2.ocaml.org#153]
A page discussing the various build tools (`make` with `automake` with [ocaml.m4](http://forge.ocamlcore.org/docman/view.php/69/53/ocaml.m4.html), `ocamlbuild`, `omake`, `oasis`, `obuild`, `jenga`,...) with pro and cons of each of them would be nice.

**Comment by agarwal** [comment]
We have the [dev_tools](http://ocaml.org/dev_tools.html) page, which needs help. I'll close #14, which is less specific than this request.

**Comment by Chris00** [comment]
I think this page should be splitted in smaller chunks.

**Comment by amirmc** [comment]
Build tools was an area that Martin (@mk270) was interested in writing about.  Not sure if he's had time yet.

**Comment by mk270** [comment]
I have indeed done some work on this, in a branch called update-devtools in mk270/ocaml.org

https://github.com/mk270/ocaml.org/tree/update-devtools

If Github's Network visualisation weren't being unduly clobbered by the autobuilder, this would be easier for people to discover by inspection.

**Comment by ghost** [comment]
Ocaml platform might supersede this now? If so, is it okay if we close this issue? cc: @patricoferris

**Comment by patricoferris** [comment]
> Ocaml platform might supersede this now? If so, is it okay if we close this issue? cc: @patricoferris

Yes I think it will cover this, and certainly the "workflows" will in the future although only for **recommended** building practices. 

One thing that has been discussed a few times is a better "history" section (cc @avsm), although I think this was more about the history of the site. Perhaps there's room to document old tools too :))?
